Modules/QuestieMenu.lua

```diff
@@ -22,7 +22,7 @@ local l10n = QuestieLoader:ImportModule("l10n")
 ---@type QuestieCorrections
 local QuestieCorrections = QuestieLoader:ImportModule("QuestieCorrections")
 
-local LibDropDown = LibStub:GetLibrary("LibUIDropDownMenuQuestie-4.0")
+local LibDropDown = LibStub:GetLibrary("LibUIDropDownMenu-4.0")
```

Comment:
We decided to keep the renaming. So this needs to be reverted

Reply:
Done again in https://github.com/Questie/Questie/pull/3154/commits/3b7d8b3c744ed71d16d939cd556fd4988393debb